src/preview.ts

```diff
@@ -0,0 +1,62 @@
+/* This file contains functions relevant for Snooty Preview */
+
+import * as vscode from 'vscode';
+import { LanguageClient } from 'vscode-languageclient';
+import { startWebview } from "./webview";
+
+// Registers Snooty Preview as a usable command
+export function registerSnootyPreview(client: LanguageClient, context: vscode.ExtensionContext): void {
+	const snootyPreview: vscode.Disposable = vscode.commands.registerCommand('snooty.snootyPreview', async () => {
```

Comment:
[nit] The formatting/tabs here look different from the rest of the files. Maybe this is automated/correct but thought I'd ask!

Reply:
Reformatted files! Not sure why the tab spaces were that big. Thanks for bringing it into attention